.env.local.example

```
@@ -0,0 +1,9 @@
+# Copy to .env.local and fill in values.
+# .env.local is gitignored — never commit credentials.
+#
+# Usage: make run-sample-remote
+
+MLFLOW_TRACKING_URI=https://your-mlflow-instance.example.com/mlflow
+MLFLOW_AUTH_TOKEN=sha256~your-token-here
+MLFLOW_WORKSPACE=your-namespace
+MLFLOW_INSECURE_SKIP_TLS_VERIFY=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MLFLOW_INSECURE_SKIP_TLS_VERIFY key should go before the MLFLOW_TRACKING_URI key

(UnorderedKey)